var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -187,11 +187,15 @@ def cmake_args(self):
             self.define_from_variant("WarpX_QED_TABLE_GEN", "qedtablegen"),
         ]
 
+        with when("+mpi"):
```

Comment:
Please add a comment, this is odd and should read it's cray specific

Reply:
It is actually occasionally required on non-cray systems. It just happened to show up on a cray system this time.